site/sfguides/src/intro_to_snowpark_container_services/intro_to_snowpark_container_services.md

```diff
@@ -0,0 +1,544 @@
+CREATE ROLE SPCS_USER_ROLE;
+GRANT CREATE DATABASE ON ACCOUNT TO ROLE SPCS_USER_ROLE;
+GRANT CREATE WAREHOUSE ON ACCOUNT TO ROLE SPCS_USER_ROLE;
+GRANT CREATE COMPUTE POOL ON ACCOUNT TO ROLE SPCS_USER_ROLE;
+GRANT CREATE INTEGRATION ON ACCOUNT TO ROLE SPCS_USER_ROLE;
+GRANT MONITOR USAGE ON ACCOUNT TO  ROLE  SPCS_USER_ROLE;
+GRANT IMPORTED PRIVILEGES ON DATABASE snowflake TO ROLE SPCS_USER_ROLE;
+
+// Grant SPCS_USER_ROLE to current user
+set USERNAME = current_user();
+grant role SPCS_USER_ROLE to user identifier($username);
+
+// Create Database, Warehouse, and Image spec stage
+USE ROLE SPCS_USER_ROLE;
+CREATE OR REPLACE DATABASE SPCS_HOL_DB;
+
+CREATE OR REPLACE WAREHOUSE SPCS_HOL_WH
+  WAREHOUSE_SIZE = XSMALL
+  AUTO_SUSPEND = 120
+  AUTO_RESUME = TRUE;
+  
+CREATE STAGE IF NOT EXISTS specs
+ENCRYPTION = (TYPE='SNOWFLAKE_SSE');
+
+CREATE STAGE IF NOT EXISTS vmounts
+ENCRYPTION = (TYPE='SNOWFLAKE_SSE');
+```
+
+Run the following SQL commands in [`01_spcs_setup.sql`](https://github.com/sfc-gh-cbaechtold/spcs-101-quickstart/blob/dev/01_spcs_setup.sql) using the Snowflake VSCode Extension OR in a SQL worksheet to create the [OAuth Security Integration](), our first [compute pool](), and our [image repository]()
+```SQL
+USE ROLE ACCOUNTADMIN;
+CREATE SECURITY INTEGRATION IF NOT EXISTS snowservices_ingress_oauth
+  TYPE=oauth
+  OAUTH_CLIENT=snowservices_ingress
+  ENABLED=true;
+
+USE ROLE SPCS_USER_ROLE;
+CREATE COMPUTE POOL IF NOT EXISTS SPCS_HOL_POOL
+MIN_NODES = 1
+MAX_NODES = 1
+INSTANCE_FAMILY = standard_1;
+
+CREATE IMAGE REPOSITORY SPCS_HOL_DB.PUBLIC.IMAGE_REPO;
+
+SHOW IMAGE REPOSITORIES IN SCHEMA SPCS_HOL_DB.PUBLIC;
+```
+- The OAuth security integration will allow us to login to our UI-based services using our web browser and Snowflake credentials
+- The compute pool is the set of compute resources on which our services will run
+- The image repository is the location in Snowflake where we will push our Docker images so that our services can use them
+
+<!-- ------------------------ -->
+## Set up your local environment
+Duration: 10
+
+### Python Virtual Environment and SnowCLI
+
+- Download and install the miniconda installer from [https://conda.io/miniconda.html](https://conda.io/miniconda.html). (OR, you may use any other Python environment with Python 3.10, for example, [virtualenv](https://virtualenv.pypa.io/en/latest/)).
+
+- Open a new terminal window, navigate to your Git repo, and execute the following commands in the same terminal window:
+
+  1. Create the conda environment.
+  ```
+  conda env create -f conda_env.yml
+  ```
+
+  2. Activate the conda environment.
+  ```
+  conda activate spcs-hol
+  ```
+
+  3. Install hatch so we can build the SnowCLI:
+  ```
+  pip install hatch
+  ```
+
+  4. Install SnowCLI:
+  ```bash
+  # the following is a 3 step command that should work for Macs
+  # cd :: Downloads; git clone; cd :: snowcli
+  cd ~/Downloads && git clone https://github.com/Snowflake-Labs/snowcli && cd snowcli
```

Comment:
You're assuming that folks have a ~/Downloads directory. Let's break out this step a little.